lib/experimental.ts

```diff
@@ -65,8 +66,10 @@ export async function distroless(
     );
   }
 
+  const imageSavePath = options?.imageSavePath || tmp.dirSync().name;
```

Comment:
is `tmp.dirSync().name;` how we used to create the tmp location before? what about the `({ unsafeCleanup: true });` part?

Reply:
Before, we were using `tmp.dirSync({ unsafeCleanup: true })`, to create it. And it returns an object with `name` (directory path) and `removeCallback` to clean it up. It was creating and cleaning everything for us. Now there is the option `imageSavePath`, and when it exists we need to create the folder, if it doesn't exist, and after clean it up. (we don't need to remove the folder as you point out above, only the image that was downloaded). 
However when there is no this option (`imageSavePath`) , we need to generate the tmp folder, the idea here is to keep the same behaviour and get only the name for this tmp folder (`tmp.dirSync().name`). it's because we're already going to create and clean it ourselves. So we don't need to use the `removeCallback`, and consequently we don't need the option `unsafeCleanup: true`.
Not sure if I could explain well.